dockerfiles/base/scripts/base/commands/cmd_info.sh

```diff
@@ -149,3 +166,21 @@ print_info() {
     done
   fi
 }
+
+cmd_network() {
+  info ""
+  info "---------------------------------------"
+  info "--------   CONNECTIVITY TEST   --------"
+  info "---------------------------------------"
+
+  info "network" "eclipse/che-ip: ${GLOBAL_HOST_IP}"
```

Comment:
we should replace "eclipse/che-ip" by the utility name
I'm also unsure if we really use ${GLOBAL_HOST_IP} (IMHO we are using ${CHE_HOST})